atest/testdata/running/try_except/invalid_try_except.robot

```diff
@@ -77,7 +77,8 @@ Multiple default EXCEPTs
     END
 
 AS not the second last token
-    [Documentation]    FAIL    EXCEPT's AS marker must be second to last.
+    [Documentation]    FAIL    EXCEPT's AS can only have one variable.
+    
```

Comment:
Would be good to split this test so that there's one test for `AS` not having a variable at all and another for it having more than one variable. Please also remove the unnecessary empty row.